### PR TITLE
Updated control logic for ERvNET LAB instance

### PR DESCRIPTION
**Pull request by TarunKrShukla**
## Description
</br>
 Brief description of the requested change.

## Checklist
- [ ] I have read the instructions mentioned in [Contribute to Code](/Contributing.md).
- [ ] I have read and understood the criteria described under [submitting changes](/Contributing.md#submitting-changes). 
- [ ] The title of the PR clearly describes the intent of the PR.
- [ ] This PR does not introduce any breaking changes to the code.

### CODE REVIEW

File: src/AzSK/Framework/Configurations/SVT/ControlSettings.json [hunk 1]
```diff
@@ -17,7 +17,11 @@
     "Max_Admin_Count": 2
   },
   "ERvNet": {
-    "ResourceLockLevel": "ReadOnly"
+    "ResourceLockLevel": "ReadOnly",
+    "WhiteListedRGs" : ["ERNetwork-LAB"],
+    "WhiteListedRemoteVirtualNetworkId" : "/subscriptions/45a7f642-6aac-4fc2-914b-0b2eeb1c7974/*",
```
Comment (VishalHaibatpure) [inline src/AzSK/Framework/Configurations/SVT/ControlSettings.json]: Are we white listing subscription Id itself? If yes this should go in internal repo?

Comment (TarunKrShukla) [reply src/AzSK/Framework/Configurations/SVT/ControlSettings.json]: ok sure will remove from here,

File: src/AzSK/Framework/Configurations/SVT/ControlSettings.json [hunk 2]
```diff
@@ -17,7 +17,11 @@
     "Max_Admin_Count": 2
   },
   "ERvNet": {
-    "ResourceLockLevel": "ReadOnly"
+    "ResourceLockLevel": "ReadOnly",
+    "WhiteListedRGs" : ["ERNetwork-LAB"],
+    "WhiteListedRemoteVirtualNetworkId" : "",
```
Comment (VishalHaibatpure) [inline src/AzSK/Framework/Configurations/SVT/ControlSettings.json]: This should be array to include if many vnet ids needs to be whitelisted

File: src/AzSK/Framework/Core/SVT/Services/ERvNet.ps1 [hunk 3]
```diff
@@ -187,12 +204,52 @@ class ERvNet : SVTIaasBase
...
+        $whiteListedaddressPrefix =  $this.ControlSettings.ERvNet.WhiteListedaddressPrefix
+        $whiteListednextHopType =  $this.ControlSettings.ERvNet.WhiteListednextHopType
+      
         $subnetsWithUDRs = $this.ResourceObject.Subnets | Where-Object {$null -ne $_.RouteTable -and -not [System.String]::IsNullOrWhiteSpace($_.RouteTable.Id)}
 
         if($null -ne $subnetsWithUDRs -and ($subnetsWithUDRs | Measure-Object).count -gt 0)
         {
-			$controlResult.AddMessage([VerificationResult]::Failed, [MessageData]::new(($subnetsWithUDRs | Select-Object Name, RouteTableText)));
-			$controlResult.SetStateData("UDRs found on any Subnet of ERVNet", $subnetsWithUDRs);
+            $nonCompliantSubnetsWithUDRs = @()
```
Comment (VishalHaibatpure) [inline src/AzSK/Framework/Core/SVT/Services/ERvNet.ps1]: Code comments missing

File: src/AzSK/Framework/Configurations/SVT/ControlSettings.json (hunk 2 above)
Comment (TarunKrShukla) [reply src/AzSK/Framework/Configurations/SVT/ControlSettings.json]: Actually this design is based on single hub based architecture that's why using single SubId.

File: src/AzSK/Framework/Core/SVT/Services/ERvNet.ps1 (hunk 3 above)
Comment (TarunKrShukla) [reply src/AzSK/Framework/Core/SVT/Services/ERvNet.ps1]: done

**Pull request merged**